website/docs/security/authentication.md

```diff
@@ -0,0 +1,95 @@
+---
+sidebar_position: 1
+slug: /security/ # referenced in README.md
+description: authentication
```

Comment:
We can add more  details on the description here

Reply:
I added a bit more details here.